### PR TITLE
Add case-sensitivty constraints in flag table

### PR DESCRIPTION
**Pull request by ekweirui**
Currently, there is no mentioning of case-sensitivity in the user guide.

Thus this pull request aims to:
* add case-sensitivty constraints in flag table

### CODE REVIEW

**Review by cheeheng — DISMISSED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -396,11 +396,11 @@ You may add a contact to Plannit using the `add-person` command.
...
 This command will require three flags:
 
-| Field     | Flag | Constraints                                                        |
-|-----------|------|--------------------------------------------------------------------|
-| **Name**  | `n/` | Can only be non-empty string of alphanumeric characters and spaces |
-| **Email** | `e/` | Can only be of the format [`local-part@domain`](#5-glossary)       |
-| **Phone** | `p/` | Can only be 8 digits long                                          |
+| Field     | Flag | Constraints                                                                                                          |
+|-----------|------|----------------------------------------------------------------------------------------------------------------------|
+| **Name**  | `n/` | Name <ul><li>is case sensitive</li> <li>can only be non-empty string of alphanumeric characters and spaces</li></ul> |
```
Comment (teoyuqi) [inline docs/UserGuide.md:401]:
```suggestion
| **Name**  | `n/` | {::nomarkdown}Name <ul><li>is case sensitive</li> <li>can only be non-empty string of alphanumeric characters and spaces</li></ul>{:/}  |
```
The ` {::nomarkdown}...{:/}` tags are needed for Github pages to display html in tables correctly.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -422,10 +422,10 @@ words, an association between a person and a module will be created.
 
 This command will require two flags:
 
-| Field           | Flag | Constraints                                                        |
-|-----------------|------|--------------------------------------------------------------------|
-| **Module Code** | `m/` | Can only be non-empty string of alphanumeric characters            |
-| **Name**        | `n/` | Can only be non-empty string of alphanumeric characters and spaces |
+| Field           | Flag | Constraints                                                                                                          |
+|-----------------|------|----------------------------------------------------------------------------------------------------------------------|
+| **Module Code** | `m/` | Module code <ul><li>is non-case sensitive</li> <li>can only be non-empty string of alphanumeric characters</li></ul> |                                                             |
```
Comment (teoyuqi) [inline docs/UserGuide.md:427]:
```suggestion
| **Module Code** | `m/` | {::nomarkdown}Module code <ul><li>is non-case sensitive</li> <li>can only be non-empty string of alphanumeric characters</li></ul>{:/} |
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -422,10 +422,10 @@ words, an association between a person and a module will be created.
...
+| **Name**        | `n/` | Name <ul><li>is case sensitive</li> <li>can only be non-empty string of alphanumeric characters and spaces</li></ul> |
```
Comment (teoyuqi) [inline docs/UserGuide.md:428]:
```suggestion
| **Name**        | `n/` |  {::nomarkdown}Name <ul><li>is case sensitive</li> <li>can only be non-empty string of alphanumeric characters and spaces</li></ul>{:/} |
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -449,13 +449,13 @@ You may delete a contact from Plannit using the `delete-person` command.
 
 This command will require one flag:
 
-| Field           | Flag | Constraints                                                        |
-|-----------------|------|--------------------------------------------------------------------|
-| **Name**        | `n/` | Can only be non-empty string of alphanumeric characters and spaces |
+| Field    | Flag | Constraints                                                                                                          |
+|----------|------|----------------------------------------------------------------------------------------------------------------------|
+| **Name** | `n/` | Name <ul><li>is case sensitive</li> <li>can only be non-empty string of alphanumeric characters and spaces</li></ul> |
```
Comment (teoyuqi) [inline docs/UserGuide.md:454]:
```suggestion
| **Name** | `n/` |  {::nomarkdown}Name <ul><li>is case sensitive</li> <li>can only be non-empty string of alphanumeric characters and spaces</li></ul>{:/} |
```

File: docs/UserGuide.md [hunk 5]
```diff
@@ -469,15 +469,15 @@ command.
 
 This command will require two flags:
 
-| Field           | Flag | Constraints                                                        |
-|-----------------|------|--------------------------------------------------------------------|
-| **Module Code** | `m/` | Can only be non-empty string of alphanumeric characters            |
-| **Name**        | `n/` | Can only be non-empty string of alphanumeric characters and spaces |
+| Field           | Flag | Constraints                                                                                                          |
+|-----------------|------|----------------------------------------------------------------------------------------------------------------------|
+| **Module Code** | `m/` | Module code <ul><li>is non-case sensitive</li> <li>can only be non-empty string of alphanumeric characters</li></ul> |                                                             |
```
Comment (teoyuqi) [inline docs/UserGuide.md:474]:
```suggestion
| **Module Code** | `m/` |  {::nomarkdown}Module code <ul><li>is non-case sensitive</li> <li>can only be non-empty string of alphanumeric characters</li></ul>{:/} |
```

File: docs/UserGuide.md [hunk 6]
```diff
@@ -469,15 +469,15 @@ command.
...
+| **Name**        | `n/` | Name <ul><li>is case sensitive</li> <li>can only be non-empty string of alphanumeric characters and spaces</li></ul> |
```
Comment (teoyuqi) [inline docs/UserGuide.md:475]:
```suggestion
| **Name**        | `n/` |  {::nomarkdown}Name <ul><li>is case sensitive</li> <li>can only be non-empty string of alphanumeric characters and spaces</li></ul>{:/} |
```

**Review by teoyuqi — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (ekweirui) [reply docs/UserGuide.md:401]: Thanks for noticing this!

**Review by cheeheng — APPROVED** [review]
LGTM

**Review by teoyuqi — APPROVED** [review]
LGTM

**Pull request merged**